rskj-core/src/main/java/org/ethereum/core/TransactionExecutor.java

```diff
@@ -369,7 +369,15 @@ private void call() {
 
     private void create() {
         RskAddress newContractAddress = tx.getContractAddress();
-        cacheTrack.createAccount(newContractAddress); // pre-created
+
+        // Keep prior balance of a contract address (if any)
+        if (cacheTrack.isExist(newContractAddress)) {
+            Coin oldBalance = cacheTrack.getBalance(newContractAddress);
+            cacheTrack.createAccount(newContractAddress);
+            cacheTrack.addBalance(newContractAddress, oldBalance);
+        } else {
+            cacheTrack.createAccount(newContractAddress);
+        }
```

Comment:
Why not simply NOT create account?

Reply:
I just implemented it the same way as it's done for `create` and `create2` opcodes - https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/org/ethereum/vm/program/Program.java#L545

`Repository.createAccount` fully recreates an account from scratch. Do you thing we should simplify this logic and do nothing for existing accounts?